README.md

```diff
@@ -54,16 +54,40 @@ export interface InvocationStrategy {
   (helperName: string, label: string): boolean;
 }
 
+export interface GlobalA11yHookOptions {
+  helpers: HelperName[];
+}
+
+type HelperName =
+  | 'blur'
+  | 'click'
+  | 'doubleClick'
+  | 'fillIn'
+  | 'focus'
+  | 'render'
+  | 'scrollTo'
+  | 'select'
+  | 'tab'
+  | 'tap'
+  | 'triggerEvent'
+  | 'triggerKeyEvent'
+  | 'typeIn'
+  | 'visit';
+
+export const defaultA11yHelperNames = ['visit', 'click', 'doubleClick', 'tap'];
+
 export function setupGlobalA11yHooks(
   shouldAudit: InvocationStrategy,
-  audit: (...args: any[]) => PromiseLike<void> = a11yAudit
+  audit: (...args: any[]) => PromiseLike<void> = a11yAudit,
+  options: GlobalA11yHookOptions = { helpers: defaultA11yHelperNames }
 );
```

Comment:
I'm inclined to change the signature here. The current signature has a default parameter that allows you to invoke it without having to pass the `a11yAudit` helper.

```ts
setupGlobalA11yHooks(() => true);
```

With this change, consumers will be forced to invoke this function by always passing the `a11yAudit` function in when they want to pass `GlobalA11yHookOptions`. I think it would make sense to perhaps move the `audit` param into the `GlobalA11yHooksOptions` options object, which would allow us to pass overrides in, while still providing defaults. We'd want to instead move the defaults merge into the function body, and likely make options optional (hehe). 

It would look something like this:

```ts
export function setupGlobalA11yHooks(
  shouldAudit: InvocationStrategy,
  options?: GlobalA11yHookOptions
);
```

And then inside the function itself we'd want to merge the options with defaults. For the helpers themselves, I think we'd probably not want to merge these directly, but rather if the helpers prop was not supplied in the options, we'd want to set it to the defaults. If it is provided, we'd assume you want to control the whole thing, and have done the associated merge with the default helpers _if you want to use those helpers_. 

I should point out that this change I'm proposing would _also_ precipitate a breaking change, which would require a major version bump. That said, that's what majors are for 🤷‍♂️ .

Reply:
@scalvert What about just overloading the method signature? This way any of the following would be valid:
```
setupGlobalA11yHooks(shouldAudit)
setupGlobalA11yHooks(shouldAudit, audit)
setupGlobalA11yHooks(shouldAudit, audit, options)
setupGlobalA11yHooks(shouldAudit, options)
```
At least for the time being. Then we can merge `audit` into `options` later down the line and be much more strategic about it in terms of messaging and migration.

Reply:
That's an option too, but the need that would precipitate those overloads would be the fact that when trying to widen signatures with multiple optional parameters, the awkwardness is because we don't have an options object to begin with. If we'd had one, we could easily bolt in additional options into that object.

I would also add that the primary reason for my adding the `audit` param was for testing, and secondarily for consumers to be able to supply their own audit implementation that would likely wrap the existing `a11yAudit`. I'm not sure how common the latter is, but the migration from a param to a param in an options object is quite a simple one, so I'm not sure that presents much of a burden to consumers.

I'm not sure it makes sense to me to avoid moving this into an options object just because we want to avoid a major version bump.

Reply:
While I find both arguments persuasive, I went with overloading `setupGlobalA11yHooks` as @drewlee proposed. Feels like this will make this change easier to land without the additional overhead of announcing a breaking change.

Reply:
(Note: I think this means we can remove the `breaking` label)

Reply:
@scalvert I'm not so much concerned about this particular change necessitating a major release, but the fact that we decided to deprecate the `a11yAuditIf` function with v5.0.0. We're using it all over the place and don't yet have a plan for replacement. So I don't think that many consumers (if any) are leveraging the custom `audit` invocation, but quite a lot are using `a11yAuditIf` directly.

Reply:
@drewlee makes sense. Specifically, the replacement is to compose the same functionality together and supply that as the `audit` param in `setupGlobalA11yHooks`. If you want to use it directly, you just write your own helper in your own app, as it was always intended for apps to have their own helpers that could be overridden at the app level.

@jgwhite the thing I don't like about this overload is the need to pass in the audit function in the `setupGlobalA11yHooks` call. That said, I can live with this being temporary so long as we circle back and fix it up for 5.0.0. Thanks for doing all this work!

Reply:
@scalvert with the overloading approach you shouldn't need to pass `audit`; `setupGlobalA11yHooks(auditIf, options)` should work just fine.

Reply:
> Thanks for doing all this work!

@scalvert my pleasure! The introduction of the global hooks system was a dream come true, so thank *you* for that!

Reply:
> @scalvert with the overloading approach you shouldn't need to pass `audit`; `setupGlobalA11yHooks(auditIf, options)` should work just fine.

Oh derp, you're right. I completely overlooked that. Thanks!